### PR TITLE
add relative links in md

### PR DESCRIPTION
**Pull request by isabelizimm**
## Related Issues and Dependencies

meteor [#25](https://github.com/AICoE/meteor/issues/25)

## This introduces a breaking change

- [ ] Yes
- [X] No

## This Pull Request implements

changes links in README and applicable docs to be relative within the repo

CC: @MichaelClifford

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,29 +75,29 @@ all the resources needed.
```
Comment (oindrillac) [inline README.md:75]: Pre-requisities should this also be relative?

**Review by oindrillac — CHANGES_REQUESTED** [review]
thanks for making these changes! 🥳

Can we also modify the links to [application](https://github.com/thoth-station/elyra-aidevsecops-tutorial/blob/master/wsgi.py), [DeploymentConfig](https://github.com/thoth-station/elyra-aidevsecops-tutorial/blob/master/manifests/base/deploymentconfig.yaml) in `docs/source/model-deployment/flask-application.md` ?

File: README.md (hunk 1 above)
Comment (MichaelClifford) [reply README.md:75]: We should add it for now :+1:

**Comment by MichaelClifford** [comment]
This PR will also require rebuilding the jupyterbook html files, right?

**Comment by isabelizimm** [comment]
> This PR will also require rebuilding the jupyterbook html files, right?

Yes! I made updates to `README.md` and `docs/source/model-deployment/flask-application.md`, and rebuilt book.

**Comment by MichaelClifford** [comment]
sorry, I had just made a separate issue for rebuilding the book : ( #188 

But can you follow these [docs](https://docs.google.com/document/d/1fkLko3tG4Na_3UbGA-YzTqU-H42r-cU-skX07luyXho/edit ) to update the site? Basically, we can't have everything in `_build/` and there are some extra things in there we don't need.

**Comment by isabelizimm** [comment]
I think it will be easier to have 2 issues, so I reset to just have the md changes and no book rebuilding. :) will move next steps to #188

**Review by oindrillac — APPROVED** [review]
/lgtm 💣

**Pull request merged**